queries.sql

```diff
@@ -21,3 +21,67 @@ WHERE name != 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+COMMIT;
+
+ROLLBACK;
+
+BEGIN TRANSACTION;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
+-- Changes happend in the database
+
+
+
+BEGIN TRANSACTION;
+DELETE FROM animals
+COMMIT;
+
+-- Deleted all the animals
+
+ROLLBACK;
+
```

Comment:
- [ ] It would be nice if you rollback your command because you made your data commit means it is already saved not rollback won't work so kindly remove commit and keep rollback there :+1:
- [ ] It would be nice if After the rollback you verify if all records in the `animals` table still exist using **SELECT** as suggested by the previous reviewer :+1:

```SQL
   BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```